crates/astria-sequencer/src/app/storage/keys.rs

```diff
@@ -0,0 +1,35 @@
+pub(in crate::app) const CHAIN_ID_KEY: &str = "app/chain_id";
+pub(in crate::app) const REVISION_NUMBER_KEY: &str = "app/revision_number";
+pub(in crate::app) const BLOCK_HEIGHT_KEY: &str = "app/block_height";
+pub(in crate::app) const BLOCK_TIMESTAMP_KEY: &str = "app/block_timestamp";
+
+pub(in crate::app) fn storage_version_by_height_key(height: u64) -> Vec<u8> {
+    format!("app/storage_version/{height}").into_bytes()
```

Comment:
As for our other comments: IMO this should be a string. (also note: here `height` is using its display impl which I agree with, whereas elsewhere we convert with `to_le_bytes`, which is inconsistent).

Reply:
Changed to use strings in [11bafd5](https://github.com/astriaorg/astria/pull/1616/commits/11bafd5b4882c901267fed4909cfcadeeb669504).